source/modulo_controllers/src/RobotControllerInterface.cpp

```diff
@@ -263,20 +263,21 @@ controller_interface::return_type RobotControllerInterface::write_command_interf
       new_joint_command_ready_ = false;
     }
 
-    for (std::size_t index = 0; index < joints_.size(); ++index) {
+for (std::size_t index = 0; index < joints_.size(); ++index) {
```

Comment:
small formatting issue
```suggestion
    for (std::size_t index = 0; index < joints_.size(); ++index) {
```